dynomitemanager/src/test/java/com/netflix/dynomitemanager/monitoring/test/BlankConfiguration.java

```diff
@@ -0,0 +1,329 @@
+package com.netflix.dynomitemanager.monitoring.test;
+
+import java.util.List;
+
+import com.netflix.dynomitemanager.sidecore.IConfiguration;
+
+/**
+ * Blanck IConfiguration class used for tests.
+ * 
+ * @author diegopacheco
+ *
+ */
+public class BlankConfiguration implements IConfiguration {
```

Comment:
Are we adding a BlankConfiguration in order to validate if the code will break in the tests?